options.go

```diff
@@ -195,11 +217,16 @@ func defaultHashRingOptions(r *Ringpop) error {
 	return HashRingConfig(defaultHashRingConfiguration)(r)
 }
 
+func defaultSWIMLevel(r *Ringpop) error {
```

Comment:
If we use a single config struct for the logging config, we can just change this to `defaultLogConfiguration` and include logging config for all loggers (not just swim).